site/sfguides/src/data_engineering_pipelines_with_snowpark_python/data_engineering_pipelines_with_snowpark_python.md

```diff
@@ -57,22 +57,11 @@ You will learn about the following Snowflake features during this Quickstart:
 ### What You’ll Need
 You will need the following things before beginning:
 
-* Snowflake
+* Snowflake account
     * **A Snowflake Account**
     * **A Snowflake user created with ACCOUNTADMIN permissions**. This user will be used to get things setup in Snowflake.
     * **Anaconda Terms & Conditions accepted**. See Getting Started section in [Third-Party Packages](https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-packages.html#getting-started).
-* Miniconda
```

Comment:
How cool does it feel to remove this portion! 😎

Reply:
So good :)